dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPI.java

```diff
@@ -15,6 +15,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.dotmarketing.portlets.templates.model.Template;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.portlets.templates.model.Template'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)